packages/concerto-core/lib/modelmanager.js

```diff
@@ -66,10 +66,10 @@ class ModelManager {
     addRootModel() {
         this.addModelFile( `namespace concerto
         abstract concept Concept {}
-        abstract asset Asset {}
-        abstract transaction Transaction {}
-        abstract participant Participant {}
-        abstract event Event {}`, 'concerto.cto');
+        abstract concept Asset {}
```

Comment:
I don't think we want to convert all these to Concepts - they should retain their asset etc stereotypes.

Reply:
The are concepts: the new type hierarchy has Assets, Participant, Transaction and Event extensions of Concept. This could be written in two ways:
1. as above
2. as `abstract transaction Transaction extends Concept {}`

I don't think this makes any difference (recall no instance can be created for those since they are abstract) but with the definition nb 2. I was having some weird errors, possibly due to the fact that we are using `transaction` before it's actually defined.